core/jazz_ui/src/app/pages/environment-overview/env-overview-section.component.html

```diff
@@ -28,10 +28,7 @@
 		</div>
 	</div>
 </div>
-<div class='relative flex-col flex-grow'>
- <div class="row-detail">
-  <div class="col-detail">
-   <div class='service-details row env-details bottom-10' *ngIf='environmnt.endpoint != undefined && !isLoading && !envResponseError' >
+<div class='service-details row env-details bottom-10' *ngIf='environmnt.endpoint != undefined && !isLoading && !envResponseError' >
```

Comment:
Please add the output of the AOT build command (found in `package.json`) in all client-related PRs in the interim.

Reply:
ng serve --env=oss